shell/client/shell.html

```diff
@@ -546,6 +546,11 @@ <h4 class="who-has-access">Who has access</h4>
     </div>
     {{#if completionState.clear}}
     <div class="button-container" role="presentation">
+      {{#with invitationExplanation}}
```

Comment:
@zarvox: Both of your suggestions act subtly differently from the code that I've written. Note that `invitationExplanation` can be null (e.g. for dev users). In that case, `{{#with invitationExplanation}}<p>...</p>{{/with}}` does not render the `<p>` element at all --- it's as if the `{{#with}}` is wrapped in an `{{#if}}`. In contrast, with the variable binding that you suggest, the `<p>` is always rendered.

Note that we do use this pattern [elsewhere](https://github.com/sandstorm-io/sandstorm/blob/8a001ccf19cf59d9b484fd7f989d8e5e7e48be48/shell/packages/sandstorm-accounts-ui/account-settings.html#L191-L200).